src/components/Game.js

```diff
@@ -64,6 +66,82 @@ function Game() {
     localStorage.setItem("contenttype", "Paragraph");
     localStorage.setItem("isfromresult", "learn");
     checkVoice(voiceText, Story[storyLine]);
+    generateResponseEvent(voiceText)
+  }
+
+
+  function generateResponseEvent(voiceText){
+   
+    let texttemp = voiceText.toLowerCase();
+    const studentTextArray = texttemp.split(" ");
+    let tempteacherText = localStorage.getItem("contentText")?.toLowerCase();
```

Comment:
use Story[storyLine] value instead of localstorage value

Reply:
Removed